ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -159,48 +161,66 @@
             : routerConfig.routerPutRequestParallelism;
         crossColoEnabled = false;
         break;
       case DeleteOperation:
-        diskSuccessTarget = routerConfig.routerDeleteSuccessTarget;
-        diskParallelism = routerConfig.routerDeleteRequestParallelism;
+        diskReplicaSuccessTarget = routerConfig.routerDeleteSuccessTarget;
+        diskReplicaParallelism = routerConfig.routerDeleteRequestParallelism;
         crossColoEnabled = true;
         eligibleReplicas = getEligibleReplicas(partitionId, null,
             EnumSet.of(ReplicaState.BOOTSTRAP, ReplicaState.STANDBY, ReplicaState.LEADER));
         break;
       case TtlUpdateOperation:
-        diskSuccessTarget = routerConfig.routerTtlUpdateSuccessTarget;
-        diskParallelism = routerConfig.routerTtlUpdateRequestParallelism;
+        diskReplicaSuccessTarget = routerConfig.routerTtlUpdateSuccessTarget;
+        diskReplicaParallelism = routerConfig.routerTtlUpdateRequestParallelism;
         crossColoEnabled = true;
         eligibleReplicas = getEligibleReplicas(partitionId, null,
             EnumSet.of(ReplicaState.BOOTSTRAP, ReplicaState.STANDBY, ReplicaState.LEADER));
         break;
       case UndeleteOperation:
-        diskParallelism = routerConfig.routerUndeleteRequestParallelism;
+        diskReplicaParallelism = routerConfig.routerUndeleteRequestParallelism;
         crossColoEnabled = true;
         eligibleReplicas = getEligibleReplicas(partitionId, null,
             EnumSet.of(ReplicaState.BOOTSTRAP, ReplicaState.STANDBY, ReplicaState.LEADER));
         // Undelete operation need to get global quorum. It will require a different criteria for success.
         // Here set the success target to the number of eligible replicas.
-        diskSuccessTarget = eligibleReplicas.size();
+        diskReplicaSuccessTarget = eligibleReplicas.size();
         break;
       default:
         throw new IllegalArgumentException("Unsupported operation: " + routerOperation);
     }
-    if (diskParallelism < 1 || cloudParallelism < 1) {
+    if (diskReplicaParallelism < 1 || cloudReplicaParallelism < 1) {
       throw new IllegalArgumentException(
-          "Parallelism has to be > 0. diskParallelism=" + diskParallelism + ", cloudParallelism=" + cloudParallelism
-              + ", routerOperation=" + routerOperation);
+          "Parallelism has to be > 0. diskParallelism=" + diskReplicaParallelism + ", cloudParallelism="
+              + cloudReplicaParallelism + ", routerOperation=" + routerOperation);
     }
 
     // Order the replicas so that local healthy replicas are ordered and returned first,
     // then the remote healthy ones, and finally the possibly down ones.
     List<? extends ReplicaId> replicas =
         routerConfig.routerGetEligibleReplicasByStateEnabled ? eligibleReplicas : partitionId.getReplicaIds();
+    // In a case where a certain dc is decommissioned and blobs previously uploaded to this dc now have a unrecognizable
```

Comment:
Minor: please add blank lines before and after this new section for readability.